Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/bbox.h

```diff
@@ -54,6 +57,14 @@ namespace CGAL {
     *                           `Construct_bbox_3` must provide the functor `Bbox_3 operator()(Point_3)`
     *                           where `%Point_3` is the value type of the vertex point map.}
     *   \cgalParamNEnd
+    *
+    *   \cgalParamNBegin{bbox_scaling}
+    *     \cgalParamDescription{a double used to scale the bounding box.
+    *       The default value is 1 and the bounding box is the smallest possible
+    *       axis-aligned bounding box.}
```

Comment:
What does the latter part of the sentence mean?

Reply:
Is this better?
"The default value is 1, corresponding to smallest possible axis-aligned bounding box."